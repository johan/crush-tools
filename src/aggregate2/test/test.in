Text-1	Text-2	Numeric-1	Numeric-2	Numeric-3
first text value	a	1	8	3.14
first text value	a	1	5	4.0
first text value	b	1	3	10
second text value	b	2	2	13.333
second text value	c	2	3	6.9
second text value	c	2	1	8
second text value	d	2	7	1
