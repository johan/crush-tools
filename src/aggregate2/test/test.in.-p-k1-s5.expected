Text-1	Numeric-3
first text value	17.14
second text value	29.233
